src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -124,14 +127,15 @@
 
 
 
-
   /**
    * Launches the "main" TripleA gui enabled game client.
    * No args will launch a client, additional args can be supplied to specify additional behavior.
    * Warning: game engine code invokes this method to spawn new game clients.
    */
   public static void main(final String[] args) {
-    ErrorConsole.getConsole();
+    if (!ClientContext.gameEnginePropertyReader().useJavaFxUi()) {
+      ErrorConsole.getConsole();
+    }
```

Comment:
I'm assuming the same thing as well...
I'm currently trying to come up with a good logging system when completely moving to javafx.
If we open a dialog using JavaFX whenever an uncaught exception occurs, we might not be able to display exceptions thrown inside the Application Thread (dedicated JFX UI Thread).
Minecraft actually "solved" this problem by using a logging framework (log4j AFAIK), and "capturing" the logs with a completely different process, so if Minecraft crashes due to a OutOfMemory or similar severe error the exceptions still gets handled and a crash report gets created